server/app/forms/QuestionForm.java

```diff
@@ -192,4 +198,46 @@ public final boolean isUniversal() {
   public final void setIsUniversal(boolean universal) {
     this.isUniversal = universal;
   }
+
+  public final ImmutableSet<PrimaryApplicantInfoTag> primaryApplicantInfoTags() {
+    return this.primaryApplicantInfoTags;
+  }
+
+  // The names of these functions must match up to the fieldName value in the
+  // PrimaryApplicantInfoTag
+  public final void setPrimaryApplicantDob(boolean val) {
+    setTagState(PrimaryApplicantInfoTag.APPLICANT_DOB, val);
+  }
+
+  public final void setPrimaryApplicantEmail(boolean val) {
+    setTagState(PrimaryApplicantInfoTag.APPLICANT_EMAIL, val);
+  }
+
+  public final void setPrimaryApplicantName(boolean val) {
+    setTagState(PrimaryApplicantInfoTag.APPLICANT_NAME, val);
+  }
+
+  public final void setPrimaryApplicantPhone(boolean val) {
+    setTagState(PrimaryApplicantInfoTag.APPLICANT_PHONE, val);
+  }
+
+  private void setTagState(PrimaryApplicantInfoTag primaryApplicantInfoTag, boolean isSet) {
+    boolean present = this.primaryApplicantInfoTags.contains(primaryApplicantInfoTag);
```

Comment:
(consider) Changing `present` to `alreadyInTagList` or something to provider more context.